src/SimpleRewriter.php

```diff
@@ -53,22 +53,33 @@ public static function rewriteFileContents( string $file_contents ) : string
 
         $wordpress_site_url = untrailingslashit( $wordpress_site_url );
         $destination_url = untrailingslashit( $destination_url );
+        $destination_url_rel = URLHelper::getProtocolRelativeURL( $destination_url );
+        $destination_url_rel_cslashes = addcslashes( $destination_url_rel, '/' );
 
-        $search_patterns = [
-            $wordpress_site_url,
-            URLHelper::getProtocolRelativeURL( $wordpress_site_url ),
-            addcslashes( URLHelper::getProtocolRelativeURL( $wordpress_site_url ), '/' ),
-        ];
-        $replace_patterns = [
-            $destination_url,
-            URLHelper::getProtocolRelativeURL( $destination_url ),
-            addcslashes( URLHelper::getProtocolRelativeURL( $destination_url ), '/' ),
+        $replace_pairs = [
```

Comment:
Still prefer `patterns` than `pairs` or could go with `replacement_pattern_pairs`?

Reply:
That's fine.